applications/CoSimulationApplication/custom_data_structure/pyKratos/QuadElement.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function, absolute_import, division
+import math
+from numpy import *
+from .Element import *
+
+
+class Quadrilateral3D4N(Element):
+
+    def __init__(self, node_list):
```

Comment:
Don't you need to call the base class constructor in here?

Reply:
done